MC/bin/o2dpg_sim_workflow.py

```diff
@@ -313,7 +322,7 @@ def getDPL_global_options(bigshm=False,nosmallrate=False):
             signalneeds = signalneeds + [ BKGtask['name'] ]
        else:
             signalneeds = signalneeds + [ BKG_HEADER_task['name'] ]
-   SGNtask=createTask(name='sgnsim_'+str(tf), needs=signalneeds, tf=tf, cwd='tf'+str(tf), lab=["GEANT"], cpu='5.', mem='2000')
+   SGNtask=createTask(name='sgnsim_'+str(tf), needs=signalneeds, tf=tf, cwd='tf'+str(tf), lab=["GEANT"], relative_cpu=5/8, mem='2000')
```

Comment:
are you sure 5/8 is interpreted as float (and not as an int in which case it would be 0)? Better to put a dot.

Reply:
Python 3 does "true divide" by default. So that will always be a float